README.md

```diff
@@ -51,12 +51,12 @@ We are continuously adding more functionality to go-mysql-server. We support a s
 
 ## Custom functions
 
-- `IS_BINARY(blob)`: returns whether a BLOB is a binary file or not.
-- `SUBSTRING(str,pos)`, ` SUBSTRING(str,pos,len)`: return a substring from the provided string.
+- `IS_BINARY(blob)`: Returns whether a BLOB is a binary file or not.
+- `SUBSTRING(str,pos)`, `SUBSTRING(str,pos,len)`: Return a substring from the provided string.
 - Date and Timestamp functions: `YEAR(date)`, `MONTH(date)`, `DAY(date)`, `HOUR(date)`, `MINUTE(date)`, `SECOND(date)`, `DAYOFYEAR(date)`.
 - `ARRAY_LENGTH(json)`: If the json representation is an array, this function returns its size.
-- `SPLIT(str,sep)`: receives a string and a delimiter and returns the parts of the string splitted by the delimiter as a JSON array of strings.
-
+- `SPLIT(str,sep)`: Receives a string and a delimiter and returns the parts of the string splitted by the delimiter as a JSON array of strings.
```

Comment:
afaik `split` is irregular so past form is also `split`

if we named parameter `sep` so maybe we should say: 
> "...split by the separator"

Reply:
shall we change this on another PR?

Reply:
no, I just meant to say sth. like:
> Receives a string and a separator and returns the parts of the string split by the separator as a JSON array of strings.